test/node/msw-api/req/passthrough.node.test.ts

```diff
@@ -88,3 +97,35 @@ it('performs a request as-is if nothing was returned from the resolver', async (
     name: 'John',
   })
 })
+
+it.each([{ code: 204 }, { code: 205 }, { code: 304 }])(
```

Comment:
It'd be simpler to keep the array of status codes, the same you do in the other tests.